Source/WebCore/rendering/BackgroundPainter.cpp

```diff
@@ -422,7 +463,7 @@ void BackgroundPainter::paintFillLayer(const Color& color, const FillLayer& bgLa
         }
     }
 
-    if (maskImage && bgLayer.clip() == FillBox::Text) {
+    if (maskImage) {
```

Comment:
Is it worth asserting which types are allowed to have a maskImage?

Reply:
I don't think so.